actioncable/lib/action_cable/connection/base.rb

```diff
@@ -126,7 +126,8 @@ def on_message(message) # :nodoc:
       end
 
       def on_error(message) # :nodoc:
-        # ignore
+        # log errors to make diagnosing socket errors easier
+        logger.error "WebSocket error occurred: #{message}"
```

Comment:
In my opinion and experience, this shouldn't raise too often. In my specific case this log message would be used to indicate that the used socket didn't implement a method that is called on it. However, @maclover7 indicated earlier that `on_error` 'could also be called when protocol errors occur'. I would still vote to keep it using the error level.